core/server/services/url/Resources.js

```diff
@@ -183,67 +198,35 @@ class Resources {
     _onResourceUpdated(type, model) {
         debug('_onResourceUpdated', type);
 
-        this.data[type].every((resource) => {
-            if (resource.data.id === model.id) {
```

Comment:
This filtering logic is removed as we are now operating on the `dbResource` which is fetched using the same filters and logic as all the resources do during startup

Reply:
We are now fetching the resource all the time or? 🤔 
e.g. a post get's updated without relations being touched.
Can't we just add `_onResourceAttached`?

Reply:
Yes we are fetching the resource every time, we could optimize it so there's no db call (revert to the old way of handling), but then we end up with one more `_on...` handler and the old, a bit complex, logic of filtering has to stay

Reply:
> and the old, a bit complex, logic of filtering has to stay

Yeah that is a good argument to keep it as it is. This logic is ugly as hell 😁